src/notifications/screens/notifications.screen.js

```diff
@@ -187,6 +187,7 @@ class Notifications extends Component {
     markAsReadByDispatch(notification.id);
     navigation.navigate('Issue', {
       issueURL: notification.subject.url.replace('pulls', 'issues'),
+      isPR: notification.subject.type === 'PullRequest',
```

Comment:
Yeah that error message popping up gave me some grief some time ago 😢 

@andrewda is right, not sure why you're seeing the message all of a sudden as I made sure it wasn't happening in all the screens (someone may have merged in something that changes things). That workaround will actually solve it. Please feel free to fix it on this PR @lex111 I really appreciate it